assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/data/useInitialState.ts

```diff
@@ -29,7 +39,7 @@ const useInitialState = ({ ticketId }: BaseProps): StateInitializer => {
 
 	// convert to TPC price objects by adding
 	// "priceType" and "priceTypeOrder"
-	const prices = sortedPrices.map<TpcPriceModifier>((price) => {
+	const prices = [basePrice, ...sortedPrices].map<TpcPriceModifier>((price) => {
 		const priceTypes = getRelations({
 			entity: 'prices',
 			entityId: price.id,
```

Comment:
Here you first need to check if there is no base price and only add if there is none.

Reply:
done, thx